packages/compass-components/src/hooks/use-theme.tsx

```diff
@@ -48,7 +49,9 @@ const withTheme = function <
   ) => {
     const theme = useTheme();
 
-    const applyTheme = global?.process?.env?.COMPASS_LG_DARKMODE === 'true';
+    const applyTheme =
+      theme.enabled === true ||
```

Comment:
@Anemy , @gribnoysup The thinking about this change is so we don't have to hardcode the darkMode prop in compass-databases-navigation. In the old sidebar and in the connections sidebar there's now a new provider to always provide the Dark theme.

(At least until we have both light and dark themed versions of both sidebars and they follow the theme as set by the provider in home)

Then those components automatically pick up dark or light.

The easiest places to see this are the placeholders or the background circle hover effect on the action icons.

So to get the new light sidebar you only have to set the `COMPASS_SHOW_NEW_SIDEBAR` feature flag and not use it in conjunction with the darkmode one. And the changes we have to make when this feature lands should also be smaller.

Reply:
Love this change, makes the theme provider already useful and reduces the amount of clean up we will need to do after new sidebar or leafygreen based darkmode will roll out 👍